_maps/shuttles/independent/independent_kilo.dmm

```diff
@@ -179,7 +181,10 @@
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
 	dir = 5
 	},
-/turf/open/floor/plasteel/dark,
+/obj/effect/turf_decal/corner_techfloor_grid{
+	dir = 10
+	},
+/turf/open/floor/hangar/plasteel,
```

Comment:
You seem to have accidently used hangar tiles on all three ships. Those have planetary atmos and probably shouldn't be on the ships.